### PR TITLE
[Indigo] fix joint_states for world

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

**Comment by fmessmer** [comment]
merge together with: 
- [schunk_modular_robotics#129](https://github.com/ipa320/schunk_modular_robotics/pull/129)
- [cob_robots#276](https://github.com/ipa320/cob_robots/pull/276)
- [cob_command_tools#83](https://github.com/ipa320/cob_command_tools/pull/83)
- [cob_common#144](https://github.com/ipa320/cob_common/pull/144)
- [cob_control#25](https://github.com/ipa320/cob_control/pull/25)
- [cob_simulation#78](https://github.com/ipa320/cob_simulation/pull/78)

**Comment by floweisshardt** [comment]
visual review: OK

**Comment by fmessmer** [comment]
I'm done with finalizing the PRs now....

File: cob_gazebo_worlds/CMakeLists.txt [hunk 1]
```diff
@@ -10,12 +10,16 @@ find_package(catkin REQUIRED COMPONENTS roslaunch rostest)
 roslaunch_add_file_check(launch/empty.launch)
 
 # ipa-kitchen (urdf model)
-add_rostest(test/urdf_ipa-kitchen.test)
```
Comment (floweisshardt) [inline cob_gazebo_worlds/CMakeLists.txt:13]: why did you only move `add_rostest` and not all test related statements?

**Comment by floweisshardt** [comment]
visual review: OK

**Comment by NadiaHG** [comment]
Visual Review: OK

**Pull request merged**